endorsement/endorsement.go

```diff
@@ -128,7 +125,7 @@ func (en *Endorsement) Proto() (*iotextypes.Endorsement, error) {
 
 // LoadProto converts a protobuf message to endorsement
 func (en *Endorsement) LoadProto(ePb *iotextypes.Endorsement) (err error) {
-	if en.ts, err = ptypes.Timestamp(ePb.Timestamp); err != nil {
+	if err = ePb.Timestamp.CheckValid(); err != nil {
```

Comment:
missing `en.ts = AsTime()`?

Reply:
this should be the reason of test failure

Reply:
Thanks for the fix, haven't notice that.